apps/vc-api/src/vc-api/workflows/workflow.service.ts

```diff
@@ -119,7 +119,7 @@ export class WorkflowService {
     // From: https://w3c-ccg.github.io/vc-api/#participate-in-an-exchange
     // "Posting an empty body will start the exchange or return what the exchange is expecting to complete the next step."
     if (presentation === undefined) {
-      return exchange.getCurrentStepRequirements();
+      return exchange.getExchangeResponse();
```

Comment:
I removed `getCurrentStepRequirements`, instead just using `getExchangeResponse`. My thinking is that the exchange response is the step requirements until data is submitted to the step. If the data is sufficient for the query, then the exchange moves to the next step.

Reply:
@jrhender 
Makes sense.

Reply:
@jrhender Makes sense.
Though, is there a possibility that the `getExchangeResponse` doesn't return the step requirement?

Reply:
Good question. I do think that if the exchange is complete and is now returning an issued VP, then `getExchangeResponse` wouldn't be returning the step requirements. But it would be returning the data that the client needs to get from the exchange and the exchange doesn't need anything more from the client, so I think it is okay. What do you think?